mess-nlesc/mess
# Model template

## About

Adding a simple script which will generate a model using a model template.

## Related issues

- #3 
- #1 

## How to test

1. Review the tutorial in README.md
1. Modify a template file to add new variables
1. Add the new variables to `variables.json` and desired values
1. run the commands below to check if new codes with desired values are generated
    ```shell
    python -m venv venv
    source ./venv/bin/activate
    ```
    
    Install the required dependencies:
    
    ```shell
    python -m pip install -r requirements.txt
    ```
    
    Generate the codes from the templates:
    
    ```shell
    python generate_model.py
    ```
